doc/theoretical_description_regression.rst

```diff
@@ -216,4 +248,8 @@ References
 ==========
 
 [1] Rina Foygel Barber, Emmanuel J. Candès, Aaditya Ramdas, and Ryan J. Tibshirani.
-"Predictive inference with the jackknife+." Ann. Statist., 49(1):486–507, February 2021.
\ No newline at end of file
+"Predictive inference with the jackknife+." Ann. Statist., 49(1):486–507, February 2021.
+
+[2] Byol Kim, Chen Xu, and Rina Foygel Barber.
+"Predictive Inference Is Free with the Jackknife+-after-Bootstrap."
+34th Conference on Neural Information Processing Systems (NeurIPS 2020).
```

Comment:
Could you add this reference also to the end of the main README ?

Reply:
Elle y est déjà dans le README.rst de ma branche